_docs.py

```diff
@@ -55,22 +57,22 @@ def update(c, language='en'):
     else:
         if not Path(target).exists():
             build(c, target=target, opts=opts)
-        c.run(f'python3 -m pipenv run sphinx-intl update -p {target} -l {language}')
-        for DIR in ['pages', 'posts', 'shop']:
-            rmtree(f'locales/{language}/LC_MESSAGES/{DIR}/')
+        c.run(
+            f'python3 -m pipenv run sphinx-intl update -p {target} -l {language}'
+        )
+        # for DIR in ['pages', 'posts', 'shop']:
```

Comment:
Yes. removing directories after the fact should not have to be done, instead they should not be there from the start.

Reply:
To prevent the `locales/` from being destroyed.

It is recommended that `invoke doc.update` be disabled. `doc/` also does not need to be localized.